lib/config/migration.spec.ts

```diff
@@ -20,7 +20,6 @@ describe('config/migration', () => {
           {
             platform: 'docker',
             endpoint: 'https://docker.io',
-            host: 'docker.io',
```

Comment:
I thought this test was *intended* to show a duplicate host. Why not needed now?

Reply:
Will update once the discussion is complete.

Reply:
The behaviour has changed from migrating silently to warning the user by throwing an error so the test isn''t valid anymore. 
Actual test would expect an `CONFIG_VALIDATION` error.